bin/nrnpyenv.sh

```diff
@@ -119,7 +119,14 @@ fi
 if test "$z" = "Darwin" ; then
   p=`$WHICH $PYTHON`
   d=`dirname $p`
-  l=`ls $d/../lib/libpython*.dylib`
+  # Get the python lib dir in an official way, working with virtualenv
+  PYLIB=$(python -c 'from distutils import sysconfig; print(sysconfig.get_config_var("LIBDIR"))')
```

Comment:
I guess I don't understand this bash syntax with a ```$(entire line)```. I'm worried about the ```python``` token after a few lines earlier where trouble was taken to specify the exact python with ```p=`$WHICH $PYTHON```. Is ```python``` really the python desired by the user if there was an argument passed to the script.